plugins/com.google.cloud.tools.eclipse.integration.appengine/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -0,0 +1,71 @@
+cleanup.add_default_serial_version_id=true
```

Comment:
What does this mean exactly? I'll need to refresh my memory of Bloch's recommendations for serialVersionUIDs.